docs/standard/memory-and-spans/index.md

```diff
@@ -16,7 +16,7 @@ Starting with .NET Core 2.1, .NET includes a number of interrelated types that r
 
 - <xref:System.ReadOnlySpan%601?displayProperty=nameWithType>, an immutable version of the <xref:System.Span%601> structure.
 
-- <xref:System.Memory%601?displayProperty=nameWithType>, a contiguous region of memory that is allocated on the managed heap rather than the stack. A <xref:System.Memory%601> instance can be backed by an array of type `T` or a <xref:System.String>. Because it can be stored on the managed heap, <xref:System.Memory%601> has none of the limitations of <xref:System.Span%601>.
+- <xref:System.Memory%601?displayProperty=nameWithType>, a wrapper over a contiguous region of memory. <xref:System.Memory%601?displayProperty=nameWithType> can also be stored on the managed heap rather than only on the stack. A <xref:System.Memory%601> instance can be backed by an array of type `T` or a <xref:System.String>. Because it can be stored on the managed heap, <xref:System.Memory%601> has none of the limitations of <xref:System.Span%601>.
```

Comment:
I don't think the inserted sentence adds clarity. The sentence at the end of this paragraph already says that `Memory<T>` instances can be stored on the heap.

Maybe the inserted sentence needs to be reworded? What point needs clarified?

Reply:
The initial point was to add something after the first sentence "a wrapper over a contiguous region of memory", as it replaced the confusing (In my opinion) "a contiguous region of memory that is allocated on the managed heap rather than the stack". After rereading it, I can see it is actually redundant. I will remove it.